src/Spice86.Core/Emulator/InterruptHandlers/Dos/DosInt2fHandler.cs

```diff
@@ -31,6 +31,7 @@ public class DosInt2fHandler : InterruptHandler {
         _dispatchTable.Add(0x15, new Callback(0x15, SendDeviceDriverRequest));
         _dispatchTable.Add(0x43, new Callback(0x43, () => ClearCFAndCX(true)));
         _dispatchTable.Add(0x46, new Callback(0x46, () => ClearCFAndCX(true)));
+        _dispatchTable.Add(0xFB, new Callback(0xFB, () => ClearCFAndCX(true)));
```

Comment:
Let's at least log what the game is trying to do, so we can go in and properly implement it later.

Reply:
The thing is, this interrupt is not documented. So In don't know what the game is trying to do.

Reply:
So we log an warning that the game is trying to call an undocumented interrupt 0xFB and that we are going to ignore it for now.

Reply:
Done.